features/api/retrive-book.feature

```diff
@@ -0,0 +1,26 @@
+@api
+Feature: Book API Existing and Non-existent Book Retrieval
+
+    Background:
+        Given the book database is empty
+        And I am authenticated with username "admin" and password "password"
+
+    Scenario: Admin can retrieve an existing book
+        And I am authenticated with username "admin" and password "password"
+        When I have created a book with following details:
+            | title      | author       |
+            | Admin Book | Admin Author |
+        When I send a "GET" request to "/api/books/{stored-id}"
+        Then the response status code should be 200
```

Comment:
Verify the response object also, like check id should be a number, check tittle and author extra.